components/pd_client/src/client.rs

```diff
@@ -192,9 +192,12 @@ impl RpcClient {
     /// Creates a new call option with default request timeout.
     #[inline]
     pub fn call_option(client: &Client) -> CallOption {
-        client
-            .inner
-            .rl()
+        Self::call_option_inner(&client.inner.rl())
+    }
+
+    #[inline]
+    fn call_option_inner(inner: &Inner) -> CallOption {
+        inner
             .target_info()
             .call_option()
```

Comment:
As `target_info` is fixed, seems we can cache it and avoid building `call_option` everytime until `target_info` is changed after reconnection. /cc @BusyJay